### DISCUSSION TITLE
Running quick fixes for the inspections automatically

### DISCUSSION THREAD

**Discussion by olivernybroe** [JetBrains/Qodana#169] [Q&A] [answered]
Is it possible to make Qodana run quick fixes for different inspections automatically? 

Many of the inspections we choose to enable could easily be auto fixed with the existing quick fixes.

**Comment by brichbash** [comment, marked as answer]
Hello,

Not yet, but it is a feature that will be available in our 2023.2 linters. 

Follow us on [Twitter](https://twitter.com/qodana) or subscribe to [our blog](https://blog.jetbrains.com/qodana/) to not miss the update.

**Reply by olivernybroe** [reply]
Interesting 👀  that is something we would be looking very forward to use!
